docs/3.0/gem-server-authentication.md

```diff
@@ -142,6 +142,39 @@ docker compose build --build-arg BUNDLE_PACKAGER__DEV=xxx
 ```
 </Option>
 
+<Option name="Kamal">
+Kamal setup is very similar to Docker: include `BUNDLE_PACKAGER__DEV` in your secrets and then use it in your `Dockerfile`.
```

Comment:
```suggestion

Kamal setup is very similar to Docker: include `BUNDLE_PACKAGER__DEV` in your secrets and then use it in your `Dockerfile`.
```

Just adding a new line, otherwise the code syntax don't apply correctly